src/EditorFeatures/TestUtilities/Completion/AbstractCompletionProviderTests.cs

```diff
@@ -626,6 +626,7 @@ private static Document WithChangedOption(Document document, OptionKey optionKey
             var service = GetCompletionService(document.Project);
             var completionList = await GetCompletionListAsync(service, document, position, RoslynCompletion.CompletionTrigger.Invoke);
             var items = completionList.Items;
+            Assert.Contains(items, i => i.DisplayText + i.DisplayTextSuffix == itemToCommit);
```

Comment:
This just makes it easier to see what items are suggested when a test fails because the expected item is not there.